pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -623,79 +620,127 @@ def copyTurbineFilesForEachCase(self, writeFiles=True):
                         assert yaw_mis_deg_ == 0
         
                     if EDmodel_ == 'FED':
+                        # from scipy.interpolate import interp1d
                         # Update each turbine's ElastoDyn
-                        self.ElastoDynFile['RotSpeed']   = self.bins.sel(wspd=Vhub_, method='nearest').RotSpeed.values
-                        self.ElastoDynFile['BlPitch(1)'] = self.bins.sel(wspd=Vhub_, method='nearest').BlPitch.values
-                        self.ElastoDynFile['BlPitch(2)'] = self.bins.sel(wspd=Vhub_, method='nearest').BlPitch.values
-                        self.ElastoDynFile['BlPitch(3)'] = self.bins.sel(wspd=Vhub_, method='nearest').BlPitch.values
-        
                         self.ElastoDynFile['NacYaw']   = yaw_deg_ + yaw_mis_deg_
                         self.ElastoDynFile['BldFile1'] = self.ElastoDynFile['BldFile2'] = self.ElastoDynFile['BldFile3'] = f'"{self.bladefilename}"'
                         self.ElastoDynFile['TwrFile']  = f'"{self.towerfilename}"'
                         self.ElastoDynFile['Azimuth']  = round(np.random.uniform(low=0, high=360)) # start at a random value
+                        # x_rot = x1*cosd(self.inflow_deg[case]) - y1*sind(self.inflow_deg[case])    
+                        # y_rot = x1*sind(-self.inflow_deg[case]) + y1*cosd(-self.inflow_deg[case])
                         if writeFiles:
                             if t==0: shutilcopy2_untilSuccessful(os.path.join(self.templatePath,self.bladefilename), os.path.join(currPath,self.bladefilename))
                             if t==0: shutilcopy2_untilSuccessful(os.path.join(self.templatePath,self.towerfilename), os.path.join(currPath,self.towerfilename))
                             self.ElastoDynFile.write(os.path.join(currPath,f'{self.EDfilename}{t+1}_mod.dat'))
+                            if self.floating == 1:
+                                self.HydroDynFile['PotFile'] = os.path.join(self.templatePath,'HydroData', f'Activefloat_T{t+1}')
+                                self.HydroDynFile.write(os.path.join(currPath, f'{self.HDfilename}{t+1}_mod.dat'))
+                                self.MoorDynFile     = self._check_and_open(self.MDfilepath+f'{t+1}.dat')
+                                for line in range(len(self.MoorDynFile['ConnectionProp'])):
+                                    x1=float(self.MoorDynFile['ConnectionProp'][line][2])
+                                    y1=float(self.MoorDynFile['ConnectionProp'][line][3])
+                                    x_rot = x1*cosd(self.inflow_deg[case]) + y1*sind(self.inflow_deg[case])    
+                                    y_rot = -x1*sind(self.inflow_deg[case]) + y1*cosd(self.inflow_deg[case])
+                                    self.MoorDynFile['ConnectionProp'][line][2]= str(round(x_rot,2))
+                                    self.MoorDynFile['ConnectionProp'][line][3]= str(round(y_rot,2))
```

Comment:
I am rotating the mooring lines. Since the wind direction cannot be changed and we are rotating the layout. The mooring anchors and fairleads should also be rotated. This is what I am doing here.